src/ticTacToe/game/Game.java

```diff
@@ -0,0 +1,293 @@
+     */
+    public void stopTheGame(){
+        gameStarted = false;
+        currentPlayer = null;
+        new DisplayPlayer().display();
+        for ( int i = 0; i < fieldSize; i++) {
+            for ( int j = 0; j < fieldSize; j++) {
+                updateField(i,j, EMPTY);
+                UserInterface.getButton(i,j).setWaitingForGame();
+            }
+        }
+    }
+
+    /**
+     *  method freezing game, calling from results checker. Used for watching the field after game finished.
+     *
+     * @see GameResult
+     */
+    public void endTheGame(){
+        gameStarted = false;
+        currentPlayer = null;
+        new DisplayPlayer().display();
+        for ( int i = 0; i < fieldSize; i++) {
+            for ( int j = 0; j < fieldSize; j++) {
+                UserInterface.getButton(i,j).setWaitingForGame();
+            }
+        }
+        System.out.println("game finished");
+    }
+
+    /**
+     * method for update field matrix with new value and current game turn
+     * changing active player, displaying it and making next move
+     *
+     * @param string string number in matrix
+     * @param row row number in matrix
+     * @param fieldValue value which should be write
+     * @see Player
+     * @see DisplayPlayer
+     */
+    public void nextMove(int string, int row, int fieldValue) {
+        updateField(string,row,fieldValue);
+        UserInterface.getButton(string,row).printFieldElement();
+
+        new GameResult().checkGameResult();
+
+        if (getActiveFigure() == CROSS) {
+            activeFigure = ZERO;
+        } else {
+            activeFigure = CROSS;
+        }
+
+        if (currentPlayer == player1) {
+            currentPlayer = player2;
+        } else {
+            currentPlayer = player1;
+        }
+
+        new DisplayPlayer().display();
+
+        if (currentPlayer != null) {
+            currentPlayer.makeMove();
+        }
+    }
+
+    /**
+     * method updates cell in field by given coordinates
+     * @param string number of string
+     * @param row number of row
+     * @param fieldValue valuethat should be written
+     */
+    public void updateField(int string, int row, int fieldValue) {
+        fieldValues[string][row] = fieldValue;
+    }
+
+    /**
+     * getters and setters for game parameters
+     */
+    public  int getActiveFigure() {
+        return activeFigure;
+    }
+
+    public  int getFieldValue(Integer string, Integer row) {
+        return fieldValues[string][row];
+    }
+
+    public  int[][] getFieldValues() {
+        return fieldValues;
+    }
+
+    public  void setFirstPlayerUserSelection(FirstPlayerSelect firstPlayerUserSelection) {
+        this.firstPlayerUserSelection = firstPlayerUserSelection;
+    }
+
+    public  boolean isGameStarted() {
+        return gameStarted;
+    }
+
+    public  Player getCurrentPlayer() {
+        return currentPlayer;
+    }
+
+    public  Player getPlayer(int id) {
```

Comment:
Use one spacebar to distinguish access modifier and the type here and in all other same places, not 2 spacebars